modules/runtime/scenario/scenario_generation/config_with_ease.py

```diff
@@ -135,11 +135,16 @@ def goal(self, world):
     """
     # should be access safe, otherwise would not reach.
     road_corr = world.map.GetRoadCorridor(self._road_ids, XodrDrivingDirection.forward)
-    lane_corr = road_corr.lane_corridors[self._lane_corridor_id]
```

Comment:
This should not be modified. You should only add the SequentialGoalDefinition in the BARK-ML scenario generation